inference-engine/src/low_precision_transformations/src/common/subgraph.cpp

```diff
@@ -86,6 +90,28 @@ bool Subgraph::fillSubgraphForQuantization(
     return true;
 }
 
+bool Subgraph::atLeastOneIsIntermediate(const std::shared_ptr<ngraph::Node>& node) const {
```

Comment:
I bet we need detailed comment what this method is actually check.

Reply:
Issue #37334 was extended